### PR TITLE
motr lib chnaged to ha lib

### PR DESCRIPTION
**Pull request by sonalk0209**
# Problem Statement
Removing dependancy on motr lib

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: libs/ha/ha_common_libs_k8s.py [hunk 1]
```diff
@@ -1701,3 +1701,19 @@ def poll_to_get_resource_status(self, exp_sts, rsc, rsc_id,
...
             return False
         return True
+
+    @staticmethod
+    def get_rc_node(node_obj):
+        """
+        To get the primary cortx node name (RC node)
+        :param node_obj: object for master node
+        :return: Primary(RC) node name in the cluster
+        :rtype: str
```
Comment (ankita-93) [inline libs/ha/ha_common_libs_k8s.py:1711]: Two return statements.

**Review by ankita-93 — APPROVED** [review]
one minor change..rest lgtm

File: libs/ha/ha_common_libs_k8s.py (hunk 1 above)
Comment (sonalk0209) [reply libs/ha/ha_common_libs_k8s.py:1711]: one is return and one rtype

**Comment by sonalk0209** [comment]
[collect_pr1486.log](https://github.com/Seagate/cortx-test/files/8580014/collect_pr1486.log)
collection log

**Comment by sonalk0209** [comment]
Ran CICD test which was breaking with this branch:
============================================================== short test summary info ==============================================================PASSED tests/ha/test_pod_failure.py::TestPodFailure::test_object_crud_pod_failure
===================================================== 1 passed, 6 warnings in 947.73s (0:15:47) =====================================================

**Review by zohebkhann — APPROVED** [review]

**Review by ApurwaMohite — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**